tools/databinding/databinding_stubs.bzl

```diff
@@ -42,6 +42,14 @@ def _is_direct(dep, tags):
             return False
     return True
 
+def _list_or_depset_to_list(list_or_depset):
+    if type(list_or_depset) == "list":
+        return list_or_depset
+    elif type(list_or_depset) == "depset":
+        return list_or_depset.to_list()
+    else:
+        return _error("Expected a list or a depset. Got %s" % type(list_or_depset))
```

Comment:
`_error` function is undefined in this target, please consider using https://docs.bazel.build/versions/main/skylark/lib/globals.html#fail